Feature/1 edit config (#2)
cloudogu/kubectl-ces-plugin: docs/development/development_en.md

# Developing `kubectl-ces`

## Encouraged Command Structure

The used CLI framework [github.com/spf13/cobra](https://github.com/spf13/cobra) provides a handy convention on how to structure a CLI commands:
> APPNAME VERB NOUN --ADJECTIVE

Anyhow, this plug-in will most likely deal with a large enough bandwidth of use cases for this plugin to enhance this command structure. To provide a matching narrative for different use cases the structure adds a GROUP command:
> kubectl ces GROUP VERB NOUN --ADJECTIVE

> This enables to keep an easily understandable command structure and enables use cases like these (some may not be implemented yet):

Author: ```suggestion
This still keeps the command structure easily understandable and enables use cases like these (some may not be implemented yet):
```
